### ISSUE TITLE
WCAG2Mobile: Draft feedback - 2.4.5 Multiple Ways

### ISSUE THREAD

**Issue by ChrisLoiselle** [w3c/wcag2ict#668]
Mary Jo: 

- No comment for now, but will need to watch progress on issue #13.

Chris L 

- Discussion mentions "set of app screens" that can stand alone from the "set of apps", detail goes into possibly application within a subgroup of set apps screens. 
- Example that may apply per heavy content is a music app. Example was provided on browse in multiple ways, artists, albums, genres. So in this context, multiple ways would apply and should be written to allow for that affordance. 
- Will continue to watch the discussion.
- https://www.w3.org/TR/wcag2mobile-22/#success-criterion-2-4-5-multiple-ways
- [Read issue #13 on GitHub](https://github.com/w3c/matf/issues/13)

**Comment by maryjom** [comment]
Closing since there is not yet any text proposed in the editor's draft of the WCAG2Mobile guidance.
